docs/ReferenceGuides/UnderscoredAttributes.md

```diff
@@ -449,6 +449,20 @@ through WebAssembly's import interface.
 
 It's the equivalent of clang's `__attribute__((import_module("module"), import_name("field")))`.
 
+### `@_extern(c, [, <"cName">])`
+
+Indicates that a particular declaration should refer to a
+C declaration with the given name. If the optional "cName"
+string is not specified, the mangled Swift name will be used.
```

Comment:
This seems _surprising_. I would expect
```
@_extern(c)
func malloc(_ size: UInt) -> UnsafeRawPointer?
```
to call a function called `malloc`, not `$s4tst26mallocySVSgSuF`.

Reply:
I used the naming strategy to keep the same behavior with `@silgen_name`. But on second thought, using names without Swift mangling scheme seems more natural since no one defines C function with a mangled Swift name 👍